_data/projects/academico.yml

```diff
@@ -0,0 +1,11 @@
+name: Academico
+desc: A simple, Laravel-based school management platform
+site: https://academico.site
+tags:
+  - laravel
+  - php
+  - lms
+  - vuejs
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/laxsmi/academico/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```

Comment:
@laxsmi Please change this link to https://github.com/laxsmi/academico/labels/up-for-grabs

Reply:
@ritwik12 Thank you, I just made the change you requested.